### DISCUSSION TITLE
Dynamic Hint

### DISCUSSION THREAD

**Discussion by ZevedoAFT** [filamentphp/filament#13553] [Help] [answered]
### Package

Form builder

### Package Version

V3.2

### How can we help you?

Is it possible to show a loader icon (similar to the icon that shows when a hint is clicked) has a hint of a select field only when the options are being populated?
For example: I have a TextInput that is live and when it is changeg, the options of the select field are populated by returning a controller result that could take a couple of seconds to complete. While it waits i want to show a spinning loader has a hint on the select field.

How can I achieve that?
Thank you

**Comment by leandrocfe** [comment, marked as answer]
You can use this

```php
->hint(fn () => new HtmlString(Blade::render('<x-filament::loading-indicator class="h-5 w-5" wire:loading wire:target="data.your_field"  />')))
```

If you have `TextInput::make('name')`, you should use `wire:target="data.name"`

**Reply by ZevedoAFT** [reply]
Perfect.
Thank you Leandro
